python/ray/rllib/agents/sac/requirements.txt

```diff
@@ -0,0 +1,22 @@
+pandas==1.0.1
```

Comment:
Do we need this file any more?

Reply:
If the tests are passing in ado, then we don't need this file anymore. I thought it would be useful if someone else wanted to run it locally on their system

Reply:
I see. To me it doesn't hurt but people could run tests. probably we can keep it:)

Reply:
Alright. :)